bazel/update.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Just thinking.. does it make sense to fold this into the main envoy repo, and we can reuse the lyft/envoy-build bazel image? That image has all the stuff we need. Basically, we reduce this command down to one liner (docker run -it lyft/envoy-build -c ... )

Reply:
This script just bulids the go stuff from API repo skipping cc and py code.